docs/advanced/Enhancers.md

```diff
@@ -42,7 +42,21 @@ function beautifier(renderer) {
 }
 
 const config = {
-  plugins: [ beautifier ]
+  enhancers: [ beautifier ]
+}
+
+const renderer = createRenderer(config)
+```
+
+### Monolithic class names
```

Comment:
Guess we should remove this one here and rather rename "Hot Reloading" to "Developer Experience" and have both "Hot Reloading" and "Monolithic CSS classes" as headers.